src/clients/cli.rs

```diff
@@ -395,8 +395,6 @@ mod tests {
     }
 
     impl Image for HelloWorld {
-        type EntryPoint = std::convert::Infallible;
```

Comment:
I definitely want this patch in as-is. Maybe move it to the front of the PR so rebasing is easier :)